source/dem/output_force_torque_calculation.cc

```diff
@@ -0,0 +1,39 @@
+#include <dem/output_force_torque_calculation.h>
+
+template <int dim>
+void
+write_forces_torques_output_locally(
+  std::map<unsigned int, Tensor<1, dim>> force_on_walls,
+  std::map<unsigned int, Tensor<1, dim>> torque_on_walls)
+{
+  TableHandler table;
+
+  for (const auto &it : force_on_walls)
+    {
+      table.add_value("B_id", it.first);
+      table.add_value("Fx", force_on_walls[it.first][0]);
+      table.add_value("Fy", force_on_walls[it.first][1]);
+      table.add_value("Fz", force_on_walls[it.first][2]);
+      table.add_value("Tx", torque_on_walls[it.first][0]);
+      table.add_value("Ty", torque_on_walls[it.first][1]);
+      table.add_value("Tz", torque_on_walls[it.first][2]);
+    }
+  table.set_precision("Fx", 9);
+  table.set_precision("Fy", 9);
+  table.set_precision("Fz", 9);
+  table.set_precision("Tx", 9);
+  table.set_precision("Ty", 9);
+  table.set_precision("Tz", 9);
+
+  table.write_text(std::cout);
```

Comment:
You need to be careful, this won't work in parallel except if you call this function only from proc 0. Can you check that this is how you have done it now?

Reply:
This is how i did it :)